twine/common/db-schema.c

```diff
@@ -498,6 +498,15 @@ spindle_db_migrate_(SQL *restrict sql, const char *identifier, int newversion, v
 		}
 		return 0;
 	}
+	// Version 25 introduces a timestamp to delay the execution of the triggers
+	if(newversion == 25)
+	{
+		if(sql_execute(sql, "ALTER TABLE \"triggers\" ADD COLUMN \"earliest_activation\" TIMESTAMP DEFAULT NULL"))
```

Comment:
I think you want a different default or a post-alter update here - possibly 1970 or something far in the past because ...

```partir01=# select * from cg_test;
   name   | earliest_activation 
----------+---------------------
 Fish     | 
 Donkey   | 
 Weebles  | 
(3 rows)

Time: 0.361 ms
partir01=# select * from cg_test where earliest_activation < NOW();
 name | earliest_activation 
------+---------------------
(0 rows)

Time: 0.374 ms
```

You'll prevent everything inserted before this patch from ever activating.